spec/lib/nationbuilder/client_spec.rb

```diff
@@ -83,10 +83,21 @@
         client.call(:people, :create, params)
       end
 
-      expect(response['status_code']).to eq(201)
+      expect(client.response.status).to eq(201)
       expect(response['person']['first_name']).to eq('Bob')
```

Comment:
We'll need another release for this I think, that would be done outside the context of a PR cc: @kaip